OpenRA.Game/Game.cs

```diff
@@ -374,8 +374,8 @@ public static void InitializeMod(string mod, Arguments args)
 
 			ModData = null;
 
-			if (mod == null)
-				throw new InvalidOperationException("Game.Mod argument missing or mod could not be found.");
+			if (mod == null || !Mods.ContainsKey(mod))
+				throw new InvalidOperationException("Game.Mod argument '{0}' missing or mod could not be found.".F(mod));
```

Comment:
Can you please split this into two `if`s, one for it being null (`Game.Mod argument missing`) and the other if it's not found (`Game.Mod argument '{0}' it not a known mod`).  We now give the error `Game.Mod argument '' missing or mod could not be found.` if the argument isn't specified, which looks weird.